packages/design-system/src/components/SelectField/Select.js

```diff
@@ -310,12 +312,17 @@ class Select extends React.PureComponent {
             onChange={this.onSearchChange}
             disabled={disabled}
           />
-          <ClearButton
-            isVisible={!!selectedItemModel && !isOpen && !required}
-            ref={this.clearButtonRef}
-            clearSelectedOption={this.clearSelectedOption}
-          />
-          <MenuDownIcon width="24px" height="24px" fill="#424d57" />
+          { shouldRenderClearButton && 
```

Comment:
if you do it like that we will lose fade-in transition ;x

Reply:
however maybe we don't need it so much, dunno

Reply:
In my opinion we don't need it. As I see it makes no difference.